tests/code/test_certification.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+from peewee import DoesNotExist
+
+from app.models import mainDB
+from app.models.event import Event
+from app.models.certification import Certification
+from app.models.certificationRequirement import CertificationRequirement
+from app.models.requirementMatch import RequirementMatch
+
+from app.logic.certification import getCertRequirements, updateCertRequirements, updateCertRequirementForEvent
+
+@pytest.mark.integration
+def test_getCertRequirements():
+    allreqs = getCertRequirements()
```

Comment:
allreqs should be allReqs for readability. It could also help if instead of reqs we use requirements, so allRequirements